scripts/init-project.sh

```diff
@@ -35,5 +35,5 @@ END
 docker-compose exec web drush upwd dru_admin changeme
 
 # Add dev deps
-docker-compose exec web composer require woohoolabs/yang
-docker-compose exec web composer require php-http/guzzle6-adapter
\ No newline at end of file
+composer require woohoolabs/yang
```

Comment:
that works for me. but why is it installed in a single command? if a single command cuts it, let's use the single command.